docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md

```diff
@@ -24,6 +25,189 @@
 | [v1.7.1](Data_Dictionary_Release_Notes.md#release-with-api-v171) | March 16, 2017 |
 | [v1.3.1](Data_Dictionary_Release_Notes.md#release-with-api-v131) | September 7, 2016 |
 
+## v2.7.0
+
+* __GDC Product__: GDC Data Dictionary
+* __Release Date__: May XX, 2023
+
+### New Features and Changes
+
+* Altered `aligned_reads` Entity
+	* New property: `wgs_coverage`
+* Altered `case` Entity
+	* New required property: `disease_type`
+	* New required property: `primary_site`
+	* Changes made to `primary_site`
+		* New permissible value: `Not Applicable`
+* Altered `demographic` Entity
+	* New property: `country_of_birth`
+	* New property: `education_level`
+	* New property: `marital_status`
+* Altered `diagnosis` Entity
+	* New deprecated property: `metastasis_at_diagnosis_site`
+	* New property: `clark_level`
+	* New property: `contiguous_organ_invaded`
+	* New property: `ensat_clinical_m`
+	* New property: `ensat_pathologic_n`
+	* New property: `ensat_pathologic_stage`
+	* New property: `ensat_pathologic_t`
+	* New property: `first_symptom_longest_duration`
+	* New property: `measurement_type`
+	* New property: `melanoma_known_primary`
+	* New property: `tumor_burden`
+	* New property: `ulceration_indicator`
+	* Changes made to `sites_of_involvement`
+		* Changes made to `items`
```

Comment:
Remove the "items" line and lower the indentation level for the permissible values.  I think this is present in the modified arrays.

Reply:
Removed 'items' lines 👍